shared/src/commonMain/kotlin/org/itmodreamteam/myrest/shared/restaurant/RestaurantClient.kt

```diff
@@ -0,0 +1,21 @@
+package org.itmodreamteam.myrest.shared.restaurant
+
+import org.itmodreamteam.myrest.shared.ContentPage
+import org.itmodreamteam.myrest.shared.Pageable
+
+interface RestaurantClient {
```

Comment:
What about deleting restaurants?

Reply:
Why are you asking this like you have implemented this feature during the first milestone? :smiley: 
Actually, admin is able to hide or block any restaurant.